stack.yaml

```diff
@@ -15,4 +15,4 @@ extra-deps:
 - libsystemd-journal-1.4.4
 
 nix:
-  shell-file: scripts/nix/stack-shell.nix
+  shell-file: nix/stack-shell.nix
```

Comment:
why not use `shell.nix`? or a variant that does not start hoogle? (cf. https://github.com/input-output-hk/cardano-shell/blob/master/stack-shell.nix)

Reply:
Unfortunately Stack won't work with the wrapped ghc and ghc-pkg provided by Haskell.nix. It requires a plain GHC. So that's why there's a separate `stack-shell.nix`.

Reply:
@rvl looks like adding `Cabal` to the build env, via a package that depends on `Cabal` (eg. `hackage-db`), solve the problem. See #354 (and merge if that is ok for you).